src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int pickColor = new Random().nextInt(Colors.values().length);
```

Comment:
давай вынесем переменную Random на уровень класса, что б не создавать каждый раз новый экземпляр, когда мы вызываем метод getRandomColor(). pickColor - не очень информативное название переменной
```suggestion
        int colorIndex = new Random().nextInt(Colors.values().length);
```